src/core/geometry/qgsgeometryengine.h

```diff
@@ -110,42 +110,42 @@ class CORE_EXPORT QgsGeometryEngine
      *
      * \since QGIS 3.0 \a geom is a pointer
      */
-    virtual QgsAbstractGeometry *intersection( const QgsAbstractGeometry *geom, QString *errorMsg = nullptr ) const = 0 SIP_FACTORY;
+    virtual QgsAbstractGeometry *intersection( const QgsAbstractGeometry *geom, QString *errorMsg = nullptr, double gridSize = -1 ) const = 0 SIP_FACTORY;
```

Comment:
The QgsGeos constructor already has a "precision" argument -- there's no need for an additional argument for each of these methods

Reply:
It's slightly different.